docs/guides/decode-examples.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 2
+sidebar_label: Decode Examples
+---
+
+## Decode Bytes Examples
+
+Some of the new output variables from `functions` and `event` of the new `v2.1 Dex` needs to be decoded. Here are some propositions in order to do so: <br/>
+
+### Decode `amountsOut` from the `Swap` event
+
+The following functions return the decoded values (x, y) extracted from the encoded `bytes32` amounts. The functions are returning the results in a `tuple`.
```

Comment:
The parameter `amountsIn` is packed with the amounts in for both tokens X and Y and encoded into `bytes32`. Below is an example of how to decode them. The same is true for `amountsOut`.